e2e/services/browser.ts

```diff
@@ -95,12 +95,14 @@ export class TestBrowserService {
   }
 
   static async closeRemoteBrowser(connectParams?: ConnectRetryParams) {
+    if (TestBrowserService.#remoteBrowser == null) return;
```

Comment:
```suggestion
    // If the `start` button is not clicked, it will hang as the browser is not started
    if (TestBrowserService.#remoteBrowser == null) return;
```